pyquil/quil.py

```diff
@@ -643,16 +584,11 @@ def out(self, *, calibrations: Optional[bool] = True) -> str:
         Serializes the Quil program to a string suitable for submitting to the QVM or QPU.
         """
 
-        return "\n".join(
-            itertools.chain(
-                (dg.out() for dg in self._defined_gates),
-                (wf.out() for wf in self.waveforms.values()),
-                (fdef.out() for fdef in self.frames.values()),
-                (cal.out() for cal in self.calibrations) if calibrations else list(),
-                (instr.out() for instr in self.instructions),
-                [""],
-            )
-        )
+        # TODO: program str without calibrations
+        if calibrations:
+            return str(self._program)
+        else:
+            return str(self._program.into_simplified())
```

Comment:
I think the intent of `calibrations=False` would have been similar to (2) and "headers." 

The term "calibrations" inconsistently includes `DEFCAL [MEASURE]` or otherwise all `DEF*` instructions. I don't think this will cause trouble to change, but we can adjust if needed based on feedback.